chain-core/src/state/account.rs

```diff
@@ -19,6 +19,14 @@ use secp256k1::recovery::{RecoverableSignature, RecoveryId};
 use std::convert::{From, TryFrom};
 use std::fmt;
 
+const MAX_INPUTS_LENGTH: usize = 64;
```

Comment:
for exchange,
input length can be very large.

Reply:
it will be filtered out by size check,
so how about using very large count ?

Reply:
or no limit

Reply:
this is a limit for 1 transaction -- if more inputs are needed, one can submit more transactions? @leejw51crypto 
@calvinaco any thoughts on reasonable limits?

Reply:
very large utxo txs will be filtered by the size.
so limiting utxo count doesn't have much effect.

for the certain circumstances, for massive test,
large utxo in tx is necessary.
for stress test etc.

Reply:
we can use many txs though.